extensions/data-workspace/src/common/constants.ts

```diff
@@ -46,17 +48,21 @@ export const NewProjectDialogTitle = localize('dataworkspace.NewProjectDialogTit
 export const TypeTitle = localize('dataworkspace.Type', "Type");
 export const ProjectNameTitle = localize('dataworkspace.projectNameTitle', "Name");
 export const ProjectNamePlaceholder = localize('dataworkspace.projectNamePlaceholder', "Enter project name");
+export const ProjectName = localize('dataworkspace.projectName', "Project Name");
```

Comment:
Just for consistency maybe we can add a verb here, I see the other 2 prompts are "Select \<something\>" while this one just says "Project Name".

Reply:
![image](https://user-images.githubusercontent.com/28519865/123344919-bc916800-d509-11eb-92a1-734aca5ce9e9.png)

Reply:
either this one or the other two prompts should be updated for consistency with the capitalization